vllm/compilation/matcher_utils.py

```diff
@@ -85,10 +162,12 @@ def forward_custom(
         weight: torch.Tensor,
     ) -> torch.Tensor:
         result = torch.empty_like(input)
+        # TODO: support non-contiguous input for RMSNorm and remove this
+        input_contiguous = input.contiguous()
```

Comment:
I think this might be a ROCm-only requirement - can you guard it and also attach a GH issue link?